lms/static/scripts/frontend_apps/components/FileList.js

```diff
@@ -5,8 +5,27 @@ import propTypes from 'prop-types';
 import Spinner from './Spinner';
 import Table from './Table';
 
+/**
+ * @typedef File
+ * @prop {string} display_name - The filename
+ * @prop {string} updated_at - An ISO date or date + time string
+ */
+
+/**
+ * @typedef FileListProps
+ * @prop {File[]} files - List of file objects returned by a `listFiles` call.
```

Comment:
In the past, writing (non-code) conventions/styleguides I've followed suggest a convention of not using a full stop unless the (bulleted, e.g.) item is a full sentence, or if there is more than one sentence. This is obviously trivial, but I'd vote to remove the full stops from all of the descriptions in this block.

Reply:
I don't have strong opinions here. We're rather inconsistent at present. If we can come to a collective agreement I'm happy to go with either.

Reply:
Here's my proposal: Only need a full stop if you need one. That seems like a silly definition, but I think in practice it will feel clear...

Reply:
That seems reasonable. I've made that change.